File: classloader.php
<?php 

Class Classloader
{
  function __construct($path, $package)
  {
    $this->path = $path;
    $this->package = $package;

    $this->nodepath = "";
    if(PHP_OS == "Darwin") // Mac OS Apache can't find node without full path
      $this->nodepath = "/Users/Carl/local/bin/";

    system($this->nodepath."node Classloader.js ".$this->path." ".$this->package." 2>&1");
  }
}

header("Content-type: text/javascript; charset=utf-8");

header("Content-Encoding: gzip");
ob_start('ob_gzhandler') ;

$package = str_replace('_', '.', key($_GET));

if ($package == "")
{
  echo "console.error('Classloader Error: no package found');";
  die();
}
new Classloader("../src", $package);

?>

File: Classloader.py

import os
import sys
import subprocess

class Classloader():
  """Python implementation of server script to run the node classloader"""
  def __init__(self, path, package, classloaderpath=""):
    if package == None:
      raise Exception("Classloader Error: no package found")

    self.path = path
    self.package = package
  
    self.classloaderpath = classloaderpath

    self.nodepath = ""
    if sys.platform == "darwin": # Mac OS Apache can't find node without full path
      self.nodepath = "/usr/local/bin/"

  def nodeClassloader(self):
    command = [self.nodepath+"node", self.classloaderpath + "Classloader.js", self.path,self.package]
    process = subprocess.Popen(command, stdout=subprocess.PIPE)
    script = process.stdout.read().decode("utf-8") 
    return script

if __name__ == "__main__":
  path = "../source"
  package = None

  if len(sys.argv) > 1:
    package = sys.argv[1];

  try:
    classloader = Classloader(path, package)
    print classloader.nodeClassloader()
  except Exception, e:
    print "console.log('"+e.message+"');"